apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -309,12 +311,12 @@ export class CaseService {
     return getRulingPdfAsString(existingCase)
   }
 
-  getRequestPdf(existingCase: Case): Promise<string> {
+  async getRequestPdf(existingCase: Case): Promise<string> {
     this.logger.debug(
       `Getting the request for case ${existingCase.id} as a pdf document`,
     )
-
-    return getRequestPdfAsString(existingCase)
+    const intl = await this.intlService.useIntl(['judicial.system.test'], 'is')
```

Comment:
Leftover from testing
```suggestion
    const intl = await this.intlService.useIntl(['judicial.system.backend'], 'is')
```